azure-webapp-maven-plugin/src/it/linux-jar/cleanup.groovy

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+// Clean up resources created in test
+def clientId = System.getenv("CLIENT_ID")
+def tenantId = System.getenv("TENANT_ID")
+def key = System.getenv("KEY")
+def command = """
+    az login --service-principal -u ${clientId}  -p ${key} --tenant ${tenantId}
+    az group delete -y -n maven-webapp-it-rg-1 --no-wait
```

Comment:
`maven-webapp-it-rg-1`: the suffix `1` might be used by other cases. Let's use a unique suffix for each test cases.